webui/react/src/components/ExperimentActionDropdown.tsx

```diff
@@ -86,9 +89,29 @@ const ExperimentActionDropdown: React.FC<Props> = ({
   const ExperimentMoveModal = useModal(ExperimentMoveModalComponent);
   const ExperimentRetainLogsModal = useModal(ExperimentRetainLogsModalComponent);
   const HyperparameterSearchModal = useModal(HyperparameterSearchModalComponent);
+  const [experimentItem, setExperimentItem] = useState<Loadable<ExperimentItem> | 'loading'>(
```

Comment:
why have `'loading'` and `NotLoaded` together?

Reply:
`loading` was needed because `NotLoaded` couldn't tell us if data was empty or in the middle of loading. I'll move forward with your suggested approach instead of this approach.